fastapitableau/user_guide.py

```diff
@@ -86,7 +86,8 @@ def _tableau_usage_str(self):
         }
 
         if self.return_info.type not in tableau_funcs.keys():
-            raise TypeError("Unexpected return type: %s" % (self.return_info.type))
+            return f"Cannot determine Tableau script for return type '{self.return_info.type}'"
+            # raise TypeError(f"Cannot determine Tableau script for return type for function at '{self.path}': {self.return_info.type}")
```

Comment:
Do you want to remove this commented out code?

Reply:
I'll do that in my next PR :)

Reply:
I kept it in because it's something I want to discuss in tech sync tomorrow, and want to be able to revert if need be.